backend/app/assets/stylesheets/spree/backend.css

```diff
@@ -3,9 +3,5 @@
  * and any sub-directories. You're free to add application-wide styles to this file and they'll appear at
  * the top of the compiled file, but it's generally better to create a new file per style scope.
```

Comment:
What do you think if we take the occasion to change this misleading comment? It's not true that the manifest will `automatically include all the stylesheets available in this directory and any sub-directories`. It'll only take what is referenced in `spree/backend/spree_admin`.